integration/helpers.go

```diff
@@ -203,3 +203,50 @@ func AssertEqualWriteError(t *testing.T, expected *mongo.WriteError, actual erro
 	return assert.Equal(t, expected.Message, actualWriteErr.Message) &&
 		assert.Equal(t, expected.Code, actualWriteErr.Code)
 }
+
+// IsUnimplemented returns true if the error code is unimplemented.
```

Comment:
Do we have any agreement about negative vs positive statements for such cases? Double negatives cases often increase cognitive load (implemented == ! IsUnimplemented). E.g. we don't call functions `IsUnequal`, but we call them `IsEqual` instead. 

Is it on purpose to not call this function `IsImplemented` instead?

Reply:
Agree. Usually, I don't like it too. 
Here it's specifically true if it is not implemented. 
I understand this and it provides no mental mess for me. 
Of course, I'd change it if it's uncomfortable.

Reply:
removed the func.